woocommerce-payments.php

```diff
@@ -91,6 +91,36 @@ function wcpay_jetpack_init() {
 			'priority'      => 5,
 		]
 	);
+
+	// When only WooCommerce Payments is active, minimize the data to send back to WPcom for supporting Woo Mobile apps.
+	$jetpack_config->ensure(
+		'sync',
+		array_merge_recursive(
+			\Automattic\Jetpack\Sync\Data_Settings::MUST_SYNC_DATA_SETTINGS,
+			[
+				'jetpack_sync_modules'           =>
+					[
+						'Automattic\\Jetpack\\Sync\\Modules\\Options',
+						'Automattic\\Jetpack\\Sync\\Modules\\Full_Sync',
```

Comment:
This module is necessary for the add_action related to the upgrading process below.

Ref: p1636040957020000/1635935174.006800-slack-C02HHDEUEH2